MANIFEST.in

```diff
@@ -1,4 +1,3 @@
-recursive-include bqplot/static *.*
```

Comment:
We still need to include the nbextension with the package, don't we?

Reply:
I don't think si, because now it's not package data, but data_files, it is by default included.

Reply:
but bqplot.json is in that file, which is odd.

Reply:
Both of them should not be needed according to:
https://packaging.python.org/guides/using-manifest-in/

Reply:
Yes. I've generated the tar.gz using sdist and it seems included in the package even though it's not in the MANIFEST :+1: